components/shared/FormLayout.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Grid } from 'styled-css-grid';
+
+import SquareButton from './SquareButton';
+
+export const FormGrid = styled(Grid)`
```

Comment:
Really like how you created these shared layout items. Will really help the forms come together fast and all look consistent.